style.css

```diff
@@ -0,0 +1,578 @@
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0 auto;
+}
+
+/*section-header*/
+.header {
+    display: block;
+    overflow: hidden;
+    min-height: 568px;
+    max-height: 100vh;
+    background-color: #eff0f2;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 20px;
+}
+@media screen and (max-width: 1024px){
+    .header {
+        font-size: 18px;
+    }
+}
+@media screen and (max-width: 768px){
+    .header {
+        font-size: 16px;
+    }
+}
+@media screen and (max-width: 512px){
+    .header {
+        font-size: 14px;
+    }
+}
+.header__logo {
+    box-sizing: border-box;
+    width: 113px;
+    height: 43px;
+    margin-top: 3.25em;
+    margin-left: 11%;
+    border: 5px solid;
+    text-transform: uppercase;
+    font-size: 17px;
+    font-weight: 700;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.header__logo .capital-letter{
+    color: #b2f2ee;
+    font-size: 130%;
+}
+
+.header__description {
+    text-align: center;
+    font-weight: 700;
+    margin: 13% 0.24em 0;
+    color: #505050;
+    font-size: 2.3em;
+    text-transform: uppercase;
+    letter-spacing: 0.035em;
+    text-shadow: 3.5px 8px 6px #cbccce;
+}
+.header__description .text-font {
+    background-color: #b2f2ee;
+    padding: 0 0.22em;
+    margin: 0 -0.22em;
+    text-shadow: 3px 5px 5px #99d0cc;
+}
+@media screen and (max-width: 515px){
+    .header__description{
+        line-height: 130%;
+    }
+}
+
+.header__contact {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 9% auto 0;
+}
+.header__contact  .btn{
+    box-sizing: border-box;
+    width: 232px;
+    height: 53px;
+    font-size: 17px;
+    color: #505050;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 400;
+    letter-spacing: 3px;
+    border: 2px solid black;
+    border-radius: 2px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 300ms;
+}
+@media screen and (max-width: 550px){
+    .header__contact .btn{
+        width: 200px;
+        height: 45px;
+        font-size: 15px;
+    }
+    .header__logo{
+        margin-top: 2em;
+    }
+}
+.header__contact .btn:hover{
+    cursor: pointer;
+    background-color: #21ecde;
+}
+
+.header__image {
+    margin-left: 74%;
```

Comment:
Please try to stick to one kind of relative values most of the time. I advise to use `rem`. For very small values use `px` and etc